src/app/shared/table/actions/table-inline-save-action.ts

```diff
@@ -1,5 +1,15 @@
+import { Router } from '@angular/router';
+import { TranslateService } from '@ngx-translate/core';
+import { TableChargingStationsRebootAction } from 'app/pages/charging-stations/table-actions/table-charging-stations-reboot-action';
+import { CentralServerService } from 'app/services/central-server.service';
+import { DialogService } from 'app/services/dialog.service';
+import { MessageService } from 'app/services/message.service';
+import { SpinnerService } from 'app/services/spinner.service';
+import { ChargingStation, OCPPConfigurationStatus, OcppParameter } from 'app/types/ChargingStation';
 import { ButtonAction } from 'app/types/GlobalType';
-import { ButtonColor, TableActionDef } from 'app/types/Table';
+import { ButtonColor, ButtonType, TableActionDef } from 'app/types/Table';
+import { Utils } from 'app/utils/Utils';
+import { Observable } from 'rxjs';
 
 import { TableAction } from './table-action';
 
```

Comment:
Same remarks as for export action:
- Name it TableInlineSaveOcppParameterAction
- Move it to 'table-actions' directory
- Change the ButtonAction constant of ChargingStationButtonAction.INLINE_SAVE_OCPP_PARAMETER
- Extends the TableInlineSaveAction class (even if there is no shared code yet, at least you will share the icons, label, tooltip...) - check how it is done in other action classes.